ci/srht/freebsd_script

```diff
@@ -22,7 +22,9 @@ release/bin/zig build test-fmt
 release/bin/zig build test-behavior
 
 # This test is disabled because it triggers "out of memory" on the sr.ht CI service.
+# And hangs on event.channel.test
```

Comment:
This is a bit misleading and I don't think belongs in this PR. Because you have disabled the test, it is in fact *not* a reason that the CI script does not run std lib tests. The point of disabling the test below is so that the CI can run successfully. After this PR, the only thing stopping CI is the out of memory issues.